hoomd/md/NeighborListGPUStencil.h

```diff
@@ -69,6 +69,20 @@ class PYBIND11_EXPORT NeighborListGPUStencil : public NeighborListGPU
             m_tuner->setEnabled(enable);
             }
 
+        #ifdef ENABLE_MPI
+
+        virtual void setCommunicator(std::shared_ptr<Communicator> comm)
+            {
+            // call base class method
+            NeighborList::setCommunicator(comm);
+
+            // set the communicator on the internal cell lists
+            m_cl->setCommunicator(comm);
+            m_cls->setCommunicator(comm);
+            }
+
+        #endif
```

Comment:
Same as above comment. This implementation is the same as in `NeighborListStencil`

Reply:
`NeighborListGPUStencil` is not a subclass of `NeighborListStencil`, so `setCommunicator` is not inherited from it.
https://github.com/glotzerlab/hoomd-blue/blob/e8eeb42ecd11d5f36b8e2221c0c0ee79e34385d8/hoomd/md/NeighborListGPUStencil.h#L32